requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,59 @@
+## User Roles
+
+### Unauthorized user
+
+Unauthorized users can sign in, view list of available songs and musical instruments. 
+
+### Authorized user
+
+Authorized user can sign out, rate added music, buy music sheets, buy premium account and get access to some musical instruments.
+
+### Premium user
+
+Premium user can do the same things as an authorized user and also have access to premium instruments and synth mapping technology.
+
+### Sheets Creator
+
+Sheets creator can do the same thing as an authorized user and add music sheets.
+
+### Moderator
+
+Moderator monitors the rights of musicians and protects our community from malicious content.
+
+## User Case Diagram
+
+<p align="center">
+  <img src="/images/use_case_diagram.png" title="Use Case Diagram" alt="Use Case Diagram">
+</p>
+
+## User Story
+
+#### User story №1
+
+As sheets creator I want to see other users' sheets, so I don't duplicate existed sheets.
+
+#### User story №2
+
+As premium user I want to use synth mapping technology to quickly play my favorite melodies on the instrument.
+
+#### User story №3
+
+As an authorized user I want to have a profile, so I manage my actions in the app
```

Comment:
How does profile help to manage actions in the application?

Reply:
Sorry, we just wanted to say that having a profile gives you certain possibilities. Like sharing your music or synchronising between devices.